index.html

```diff
@@ -63,5 +64,250 @@ <h1>
+  <div class="project-title space-list">
+
+     <h2>Multi-Post <br>
+      Stories</h2>
+     
+   <ul class="top-list">
+       <li>CANOPY</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+       <li>Back End Dev</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+   </ul>
+
+  </div>
+   <p>
+       A daily selection of privately personalized 
+       reads; no accounts or 
+       sign-ups required.
+   </p>
+
+   <ul class="list-bottom">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+   </ul>
+   <div class="action">
+     <button type="button" class="button"> See Project </button>
+   </div>
+   
+</section>
+</section>
+
+<!-- mobile section  CARD 3  -->
+
+<section class="section-nav card">
+
+  <h2 class="header-mobile">
+       <img src="images/section_header.png" alt="logo">
+  </h2>
+
+<section class="section-mobile">
+
+  <div class="project-title identaton">
+
+     <h2>Tonic</h2>
+     
+   <ul class="top-list">
+       <li>CANOPY</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+       <li>Back End Dev</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+   </ul>
+
+  </div>
+   <p>
+       A daily selection of privately personalized 
+       reads; no accounts or 
+       sign-ups required.
+   </p>
+
+   <ul class="list-bottom">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+   </ul>
+   <div class="action">
+     <button type="button" class="button"> See Project </button>
+   </div>
+   
+</section>
+</section>
+
+<!-- mobile section  CARD 4  -->
+
+<section class="section-nav card">
+
+  <h2 class="header-mobile">
+       <img src="images/section_header2.png" alt="logo">
+  </h2>
+
+<section class="section-mobile">
+
+  <div class="project-title space-list">
+
+     <h2>Multi-Post <br>
+      Stories</h2>
+     
+   <ul class="top-list">
+       <li>CANOPY</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+       <li>Back End Dev</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+   </ul>
+
+  </div>
+   <p>
+       A daily selection of privately personalized 
+       reads; no accounts or 
+       sign-ups required.
+   </p>
+
+   <ul class="list-bottom">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+   </ul>
+   <div class="action">
+     <button type="button" class="button"> See Project </button>
+   </div>
+   
+</section>
+</section>
+
+<!-- About myself -->
+
+<section class="about-myself">
+<div class="about-container">
+  <h2>
+    About <br>
+    Myself
+  </h2>
+  <p>
+    Hello I’m a software developer! I can help
+     you build a product , feature or website 
+     Look through some of my work and
+      experience! If you like what you see and
+       have a project you need coded, don’t
+        hestiate to contact me.
+  </p>
+  <div class="social-media">
+    
+          <p class="social-text">LET’S CONNECT</p>
```

Comment:
[OPTIONAL] According to the project requirements, this content should be wrapped inside a `<hx>` tag.

<img width="438" alt="Screen Shot 2022-05-30 at 14 26 06" src="https://user-images.githubusercontent.com/82426682/171050507-8117044f-b87a-403c-94ce-3ce1cbf2e77c.png">